### PR TITLE
Modifying UserProfile.js

### PR DESCRIPTION
**Pull request by Spraynard**
User Profile Modification control buttons did not have any margin, making them stick together. This pull request provides an update

Previous
Screen Shot 2020-04-08 at 10 55 30 PM

After
Screen Shot 2020-04-08 at 11 31 20 PM

### CODE REVIEW

File: src/app/page/UserProfile/UserProfile.js [hunk 1]
```diff
@@ -35,14 +39,14 @@ const useStyles = makeStyles((theme) => ({
 //   };
 // }
 
-const ProfileControlButtons = ({ isEdit, saveAction, cancelAction, editAction }) => {
+const ProfileControlButtons = ({ isEdit, saveAction, cancelAction, editAction, classes }) => {
   return isEdit ? (
     <>
-      <Button size="large" onClick={saveAction}>
+      <Button size="large" onClick={saveAction} className={classes.controlButtons}>
```
Comment (azmainamin) [inline src/app/page/UserProfile/UserProfile.js:45]: Can we instead do this:
`<Grid container spacing={1}>
     <Grid item><Button/></Grid>
     <Grid item><Button/></Grid>
</Grid>`

This way, we let MUI handle all of the responsive styling.

Comment (Spraynard) [reply src/app/page/UserProfile/UserProfile.js:45]: Yes definitely thanks for the solution! I'll put up an update soon

Comment (azmainamin) [reply src/app/page/UserProfile/UserProfile.js:45]: There are also merge conflicts.

Comment (Spraynard) [reply src/app/page/UserProfile/UserProfile.js]: Alright I resolved merge conflicts and have updated code on the source branch

**Comment by Spraynard** [comment]
My apologies I have no idea where those changes in `src/app/layout/NavigationDrawer/NavigationDrawer.js` came from. I pulled in all of masters changes and that was a part of it I guess.

**Pull request merged**